app/models/shipit/task_execution_strategy.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Shipit
+  module TaskExecutionStrategy
+    class << self
+      def current
+        @strategy || default
+      end
+
+      def register(strategy)
```

Comment:
I don't really see the point of having a `register` method if there's only once active strategy. `Shipit.task_execution_strategy = ...` would do fine.

Reply:
Addressed by https://github.com/Shopify/shipit-engine/pull/1107/commits/ad50b895fa98515030aa503a022dbb82110f54a8